lms/djangoapps/badges/backends/base.py

```diff
@@ -0,0 +1,17 @@
+"""
+Base class for badge backends.
```

Comment:
Nit: This docstring sort of just restates the obvious -- would be nice if we included some more useful information about BadgeBackends here.